src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -343,28 +343,32 @@
   "item.minecolonies.scroll.colony.tip": "Colony: %s",
   "item.minecolonies.scroll.colony.none": "None, set one with sneak + right-click",
   "item.minecolonies.scroll_area_tp": "Ultrasafe colony group-teleport scroll",
-  "item.minecolonies.scroll_area_tp.tip": "Teleport yourself and nearby players to the colony",
+  "item.minecolonies.scroll_area_tp.tip": "Teleports yourself and nearby players to the colony",
   "item.minecolonies.scroll_buff": "Sacred Scroll of Regeneration",
   "item.minecolonies.scroll_buff.tip": "Applies regeneration to citizens around you, at least most of the time...",
   "item.minecolonies.scroll_guard_help": "Spatial guard reinforcement scroll",
   "item.minecolonies.scroll_guard_help.tip": "Call your guards to help you out for a limited time, wololo!",
   "item.minecolonies.scroll_highlight": "Worker-where-are-you scroll",
-  "item.minecolonies.scroll_highlight.tip": "Sneak-Use on a minecolonies building, to highlight and speed up its workers",
+  "item.minecolonies.scroll_highlight.tip": "Sneak + right-click on a hut block to highlight and speed up its worker(s)",
```

Comment:
I really don't mind either way, but I thought I'd just comment that sneak-use is more accurate from a usability perspective as some ~~strange~~ players might change the controls.

Reply:
Looking at you, Kartijan!